fuel/app/config/config.php

```diff
@@ -101,11 +101,9 @@
 	'log_date_format'  => 'H:i:s',
 
 	// provide a monolog handler
-	'log_handler_factory' => function($locals, $level)
-	{
+	'log_handler_factory' => function ($locals, $level) {
```

Comment:
Do you have a formatter or linter that's moving the braces here? The `phpcs` linter, which is run during the pre-commit checks, vocally prefers braces on the next line. This change (and the one on line 106) shouldn't be necessary.